.github/workflows/validate.yml

```diff
@@ -0,0 +1,29 @@
+name: validate
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release/*'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - validate-vendor
```

Comment:
Does this somehow automagically trigger `make validate-vendor`?

Reply:
Yes exactly. The idea in the follow-up is to add other validators to this matrix like `lint`, `authors`, etc... See https://github.com/docker/buildx/blob/a6a1a362ad951af488fc41388f41a2d52b25f791/.github/workflows/validate.yml#L23-L25

Reply:
Oh neat!